scripts/west_commands/runners/jlink.py

```diff
@@ -9,13 +9,18 @@
 import platform
 import re
 import shlex
+from subprocess import TimeoutExpired
 import sys
 import tempfile
 
-from packaging import version
 from runners.core import ZephyrBinaryRunner, RunnerCaps, \
     BuildConfiguration
-from subprocess import TimeoutExpired
+
+try:
+    from packaging import version
+    MISSING_REQUIREMENTS = False
```

Comment:
```suggestion
    ALL_IMPORTS_FOUND = True
```

Reply:
- "ALL" makes the "futureproofness" more explicit than a mere plural which puzzled me until I read further.
- REQUIREMENTS seems to be pushing the futureproofness a bit too far; do you really anticipate other requirements than imports?
- FOUND is from the UFADN

Reply:
> FOUND is from the UFADN

Can we treat that as future work? `MISSING_REQUIREMENTS` is already used elsewhere in the runners package; I am copying the pattern.

Reply:
(It also is a hint towards requirements.txt.)